src/js/overlay.js

```diff
@@ -166,7 +171,10 @@ Overlay.prototype.render = function() {
 
 	if (this.opts.heading) {
 		const heading = document.createElement('header');
+		const headingId = this.opts.heading.title.replace(' ', '-').toLowerCase();
```

Comment:
I guess you'd need a regex for that, don't think it's that important no, feel free to merge!